src/ECharts/Commands.purs

```diff
@@ -203,6 +218,9 @@ center a = set "center" $ T.pointToForeign a
 radius ∷ ∀ i. T.Radius → DSL (radius ∷ I|i)
 radius a = set "radius" $ T.radiusToForeign a
 
+singleValueRadius ∷ ∀ i. T.SingleValueRadius → DSL (radius ∷ I|i)
+singleValueRadius a = set "radius" $ T.singleValueRadiusToForeign a

```

Comment:
Same as `title`, the option `radius` accepts both a `radius` object and a array of 'radius' objects. As you can see in 
http://echarts.baidu.com/demo.html#radar
and
http://echarts.baidu.com/demo.html#radar-multiple